src/server/ftpserver.rs

```diff
@@ -467,6 +470,46 @@ where
         self
     }
 
+    /// Enables a password guessing protection policy
+    ///
+    /// Policy used to temporarily lock an account, source IP or the
+    /// combination of both, after a certain number of failed login
+    /// attempts for a certain time.
+    ///
+    /// There are different policies to choose from. Such as to lock
+    /// based on the combination of source IP + username or only
+    /// username or IP. For example, if you choose IP based locking,
+    /// multiple successive failed login attempts will block any login
+    /// attempt from that IP for a defined period, including login
+    /// attempts for other users.
+    ///
+    /// The default policy is the combination of source IP + username.
```

Comment:
In other words it will block only if both the IP and username match? Perhaps good to say explicitly. Thanks for the nice docs either way.